bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java

```diff
@@ -344,8 +364,10 @@ public boolean tryRestoreLogin(@Nullable String data, @Nullable String overloade
 
         if (intVersion > 5) {
             String accountCustomerId = scanner.nextLine();
```

Comment:
What does the new code mean/do?

If its bigger then 5 we consume it but only if its bigger then 6 we use it otherwise we ignore it?

Reply:
Exactly, yes, the reason is, version 5 have under some condition the wrong customer id stored. In this case, it will be newly retrieved by calling the amazon server and will be stored with version 6. 
In case it is already version 6, no extra call is neccessary

Reply:
I have added a comment in the code